packages-web/maps/tests/e2e/specs/events.spec.ts

```diff
@@ -0,0 +1,70 @@
+import page from "../pages/page";
+import mapPage from "../pages/baseMap.page";
+import dataGrid from "../objects/dataGrid";
+import { getMxObjectMarkerUrl } from "../../../src/utils/Data";
+
+describe("Events are handled in maps", () => {
+    describe("static locations", () => {
+        const alertValue = `Static location items (at location 1) does not support maker events. The On click can only be set to "Do nothing"`;
```

Comment:
Template string should only be used with variable interpolated

Reply:
done